index.html

```diff
@@ -213,18 +213,21 @@ <h2>Contact Me</h2>
               love to help with it.
             </p>
           </div>
-          <form action="https://formspree.io/f/mgedkqkj" method="post" autocomplete="off" class="form-block">
+          <form id="form" action="https://formspree.io/f/mgedkqkj" method="post" autocomplete="off" class="form-block">
             <div class="data1">
-              <input name="Name" type="text" placeholder="your Full name" required maxlength="30" />
+              <input id="name" name="Name" type="text" placeholder="your Full name" required maxlength="30" />
```

Comment:
- [ ] The name input field is not fetching the data correctly from the localstorage because the `name` and the `key` don't match. Kindly change `name="Name"` to `name="name"` I believe this should solve the issue 👍 
![image](https://user-images.githubusercontent.com/46839548/190854748-a5c565b6-9a0e-4a73-b26a-ea322d88c357.png)
![image](https://user-images.githubusercontent.com/46839548/190854763-2e0d3c45-0587-42e7-80ee-419c351315f9.png)